src/Raven.Server/ServerWide/SecretProtection.cs

```diff
@@ -35,6 +35,7 @@ public unsafe class SecretProtection
         private static readonly Logger Logger = LoggingSource.Instance.GetLogger<SecretProtection>("Server");
         private readonly Lazy<byte[]> _serverMasterKey;
         private readonly SecurityConfiguration _config;
+        private const int ServerCertificateUsageLimitation = 4;
```

Comment:
MaxDeveloperCertificateValidityDurationInMonths?

There is also a 2nd place in the same file that uses that value. Please use const there.

Reply:
I was worried the naming is too long...
done